integration_test/testutil_test.go

```diff
@@ -171,6 +173,13 @@ func (it *ITest) WithIndex() *ITest {
 	return it
 }
 
+// WithMigratedIndex initializes the index and performs the index migration.
```

Comment:
Add TODO for deletion, ditto migrateIndex.

Why don’t we use the env variable for behavior change in initializeIndex instead? Simple if can solve this but you added 2 methods for it. :)

Reply:
That's a good idea, I'll change it